src/test/ensemble_mapping_config_tests.cpp

```diff
@@ -250,17 +249,18 @@ TEST_F(PipelineWithInputOutputNameMappedModel, SuccessfullyReferToMappedNamesAnd
     EXPECT_EQ(response_tensor_name->getPrecision(), InferenceEngine::Precision::FP32);
 }
 
-TEST_F(PipelineWithInputOutputNameMappedModel, SuccessfullyReloadPipelineAfterAddingModelMapping) {
+// Disabled until CVS-41658 is resolved.
+TEST_F(PipelineWithInputOutputNameMappedModel, DISABLED_SuccessfullyReloadPipelineAfterAddingModelMapping) {
```

Comment:
This is still not working?

Reply:
This is not working due to CVS-41658 bug, after reload with new mapping config we get tensor map out of sync.